woocommerce/google-listings-and-ads
# Update WordPress CS to 3.0

### Changes proposed in this Pull Request:

Updates WordPress CS to version 3.0.1 and fixes CS issues across extension files.

The WordPress CS rules are slightly more sensitive about spacing so there were quite a few substitutions, but most of them could be fixed with `phpcbf` so I don't see any reason why we can't conform to the stricter rules.

I also added an exclusion for the check `WordPress.Security.EscapeOutput.ExceptionNotEscaped`. While it's a valid check, we don't output the content of exceptions directly, and at that point we don't know if the exception is going to be logged to a file or output(which makes it hard to escape). See additional details on this here: https://github.com/WordPress/WordPress-Coding-Standards/issues/2374

Files in the `bin` and `views` directories were not included when linting locally. In GitHub workflows we lint by using `vendor/bin/phpunit ./*` which includes all PHP files. So I went ahead and included those two folder so they lint locally as well.

### Upgraded packages
`composer update wp-coding-standards/wpcs --with-all-dependencies`
```
- Upgrading squizlabs/php_codesniffer (3.6.2 => 3.7.2)
- Upgrading wp-coding-standards/wpcs (2.3.0 => 3.0.1)
```


Closes #2066 

### Detailed test instructions:
1. Checkout `dev/updates-wpcs-3`
2. Run `composer install`
3. Lint PHP files `npm run lint:php && npm run lint:php-tests`
4. Confirm no errors are flagged
5. Run the unit tests locally to ensure that they don't flag any errors either

### Changelog entry
* Dev - Update WordPress CS to 3.0
